server/cli/src/cli.rs

```diff
@@ -444,6 +444,9 @@ async fn main() -> anyhow::Result<()> {
                 sub_context,
                 argument_schema_id: form_schema_json.map(|r| r.id.clone()),
                 comment: None,
+                is_custom: todo!(),
```

Comment:
I think we're probably ok to leave this as todo! as we're about to update the logic, but normally wouldn't want to have todo!() in main...